test/remote/gateways/remote_orbital_test.rb

```diff
@@ -349,113 +349,6 @@ def test_successful_purchase_with_echeck_on_next_day
     assert_false response.authorization.blank?
   end
 
-  [
-    {
-      card: {
-        number: '4112344112344113',
-        verification_value: '411',
-        brand: 'visa'
-      },
-      three_d_secure: {
-        eci: '5',
-        cavv: 'AAABAIcJIoQDIzAgVAkiAAAAAAA=',
-        xid: 'AAABAIcJIoQDIzAgVAkiAAAAAAA='
-      },
-      address: {
-        address1: '55 Forever Ave',
-        address2: '',
-        city: 'Concord',
-        state: 'NH',
-        zip: '03301',
-        country: 'US'
-      }
-    },
-    {
-      card: {
-        number: '5112345112345114',
-        verification_value: '823',
-        brand: 'master'
-      },
-      three_d_secure: {
-        eci: '5',
-        cavv: 'AAAEEEDDDSSSAAA2243234',
-        xid: 'Asju1ljfl86bAAAAAACm9zU6aqY=',
-        version: '2.2.0',
-        ds_transaction_id: '8dh4htokdf84jrnxyemfiosheuyfjt82jiek'
-      },
-      address: {
-        address1: 'Byway Street',
-        address2: '',
-        city: 'Portsmouth',
-        state: 'MA',
-        zip: '67890',
-        country: 'US',
-        phone: '5555555555'
-      }
-    },
-    {
-      card: {
-        number: '371144371144376',
-        verification_value: '1234',
-        brand: 'american_express'
-      },
-      three_d_secure: {
-        eci: '5',
-        cavv: 'AAABBWcSNIdjeUZThmNHAAAAAAA=',
-        xid: 'AAABBWcSNIdjeUZThmNHAAAAAAA='
-      },
-      address: {
-        address1: '4 Northeastern Blvd',
-        address2: '',
-        city: 'Salem',
-        state: 'NH',
-        zip: '03105',
-        country: 'US'
-      }
-    }
-  ].each do |fixture|
-    define_method("test_successful_#{fixture[:card][:brand]}_authorization_with_3ds") do
```

Comment:
This is a clever technique, but it is not appropriate to use it in a test file like this, because dynamically creating test cases like this makes it impossible to run these tests individually.